### PR TITLE
SSU-17: Create header

### PR DESCRIPTION
**Pull request by IsabellePinheiro**
TASK: https://cheesecakelabs.atlassian.net/jira/software/c/projects/SSU/boards/414?modal=detail&selectedIssue=SSU-17

Screen Shot 2022-08-29 at 10 54 02
Screen Shot 2022-08-29 at 11 53 45

### CODE REVIEW

**Review by vinibs — APPROVED** [review]

**Comment by vinibs** [comment]
Just noticed that you versioned a `.DS_Store` file. What about removing it from version control and adding it to `.gitignore`?

File: react-web/src/components/organisms/header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { BrowserRouter as Router, Link } from 'react-router-dom'
+
+import classNames from 'classnames'
+
+import { Button, ButtonVariant } from 'components/atoms'
+import {
+  Heading,
+  TypographyHeadingLevel,
+} from 'components/atoms/typography/heading'
+import { Label } from 'components/atoms/typography/label'
```
Comment (welladam) [inline react-web/src/components/organisms/header/index.tsx:10]: The idea is to use only the Typography component and add the `variant` to define the `Label, Heading, etc`.
If you want to change something, you just need to change the Typography file.

File: react-web/src/components/organisms/header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        </Router>
+      </nav>
+      <div className={styles.containerButton}>
+        <Button
+          iconLeft={<img src={Eth} alt="Eth" />}
+          variant={ButtonVariant.tertiary}
+          disabled
+          onClick={(): void => {
+            handleOnWalletPress
+          }}
```
Comment (welladam) [inline react-web/src/components/organisms/header/index.tsx:52]:
```suggestion
          onClick={handleOnLoginPress}
```

File: react-web/src/components/organisms/header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        </Button>
+
+        <Button
+          iconLeft={<img src={Weth} alt="Weth" />}
+          className={styles.loginButton}
+          variant={ButtonVariant.tertiary}
+          disabled
+          onClick={(): void => {
+            handleOnLoginPress
+          }}
```
Comment (welladam) [inline react-web/src/components/organisms/header/index.tsx:64]:
```suggestion
          onClick={handleOnLoginPress}
```

**Review by welladam — APPROVED** [review]
Niiice!

**Pull request merged**